src/elements/Tooltip.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react"
+import styled from "styled-components"
+
+import { BorderBox } from "./BorderBox"
+import { Sans } from "./Typography"
+
+const Wrapper = styled.div`
+  position: relative;
+  display: inline-block;
+`
+
+const Tip = styled(BorderBox)`
+  background: white;
+  border: none;
+  bottom: 100%;
+  box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.1);
+  left: 50%;
+  margin-bottom: 5px;
+  opacity: 0;
+  position: absolute;
+  text-align: left;
+  transform: translate(-50%);
+  transition: opacity 250ms ease-out;
+  width: 230px;
+
+  &:hover {
+    cursor: default;
+  }
+
+  &.active {
+    opacity: 1;
+
+    &:hover {
+      opacity: 0;
+    }
+  }
+`
+
+export interface TooltipProps {
+  content: string
+  small: boolean
```

Comment:
#optional To address your point about the extra whitespace when the text content doesn't fill up the whole 230px — it might be nice if Tooltip took an optional `width` prop to occasionally override the hard-coded 230.